knowledge-base/changing-sort-styling-datagrid-maui.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Changing Sort Styling in DataGrid for MAUI
+description: Learn how to customize the header styling and sort indicator appearance in DataGrid for MAUI to meet specific requirements.
+type: how-to
+page_title: Customizing Sort Styling and Indicator in DataGrid for MAUI
+slug: changing-sort-styling-datagrid-maui
+tags: datagrid, maui, sort, styling, sortindicatorcolor, sortindicatorascendingtext, sortindicatordescendingtext
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 11.0.0 | Telerik UI for .NET MAUI DataGrid | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova) | 
+
+## Description
+
+I need to change the header styling in DataGrid for MAUI when sorting is enabled. Specifically, I want the entire header to trigger sorting when clicked and customize the appearance of the sort arrows to match our design requirements.
+
+This knowledge base article also answers the following questions:
+- How to style the header of DataGrid for MAUI with custom sort indicators?
+- How to change the appearance of the sort arrows in DataGrid for MAUI?
+- How to set custom text for the sort indicators in DataGrid for MAUI?
+
+## Solution
+
+To change the header styling and customize the appearance of the sort indicator in the [DataGrid for MAUI](https://docs.telerik.com/devtools/maui/controls/datagrid/overview), follow these steps:
+
+### Styling the Header and Sort Indicator
+
+Use the `SortIndicatorColor` property to set the color of the sort indicator in the column header. Additionally, use `SortIndicatorAscendingText` and `SortIndicatorDescendingText` to define custom text or icons for the sort indicators.
+
+Example:
```

Comment:
The following sample code shows how to customize sort indicators in a Telerik .NET MAU DataGrid by changing their color and using different icons when sorting up or down.